cvat-canvas/src/typescript/interactionHandler.ts

```diff
@@ -411,15 +424,100 @@ export class InteractionHandlerImpl implements InteractionHandler {
                 }
             }
         });
+        window.addEventListener('keyup', (e: KeyboardEvent): void => {
+            if (!this.checkCursorPosition()) {
```

Comment:
Try to use ``this.interactionData?.enabled`` instead of this function

Reply:
It worked. No need to use this function